agent/tool-scripts/datalog/disk-datalog

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+interval=$1
+file=$2
+
+while true; do
+	date "+%s" >> $file
+	# Gendata.pm expects the output to be integers not strings, so we merge 
+	# Filesystem and Mount columns so that we will have details about mount 
+	# as well. Also we display df output in megabytes to have a single unit.
+	/usr/bin/df -BM | awk -F ' ' '{gsub(/\%/,"",$5);gsub(/\M/,"",$2);gsub(/\M/,"",$3);gsub(/\M/,"",$4); print $1"_"$6" "$2" "$3" "$4" "$5}' >> $file
```

Comment:
I think we want to have the file system mount point only in the column headers in the `.csv` file, but in the JSON we want to capture all of the fields emitted by `df`.

Also, can you use `-BK` instead of megabytes?